### PR TITLE
Add theme to flow component

### PR DESCRIPTION
**Pull request by gaokevin1**
## Description

Add ability to control theme from query parameter, will also need to add to env variables in Vercel for default.

## Screenshots

📺🔫 All of the UI influenced by this PR

## Must

- [ ] 📱 Responsiveness (mobile/XL resolutions)
- [ ] 🧪 Tests
- [ ] 📃 Documentation (if applicable)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -58,6 +58,10 @@ const App = () => {
 
 	const backgroundColor = urlParams.get('bg') || process.env.DESCOPE_BG_COLOR;
 
+	const theme = (urlParams.get('theme') ||
+		process.env.DESCOPE_FLOW_THEME ||
```
Comment (dorsha) [inline src/App.tsx:62]: Why light and not system?

Comment (asafshen) [reply src/App.tsx:62]: I think we should leave the default value to the SDK (which is indeed light)

so worth fallback to `undefined` 

also let fix the types like we talked today

File: src/App.tsx [hunk 2]
```diff
@@ -58,6 +58,10 @@ const App = () => {
...
+		'light') as any;
```
Comment (asafshen) [inline src/App.tsx:63]:
```suggestion
	const theme = (urlParams.get('theme') ||
		process.env.DESCOPE_FLOW_THEME) as React.ComponentProps<typeof Descope>['theme'];
```

**Review by asafshen — APPROVED** [review]
Cool

**Pull request merged**